x-pack/plugins/security/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "./target/types",
+    "emitDeclarationOnly": true,
+    "declaration": true,
+    "declarationMap": true
+  },
+  "include": ["common/**/*", "public/**/*", "server/**/*"],
+  "references": [
```

Comment:
Thanks for taking the time to sort these alphabetically 🏅

Reply:
I must admit it becomes an obsession for me :see_no_evil: